js/index.js

```diff
@@ -1,7 +1,55 @@
-// Iteration 1: Names and Input
+console.log("I'm ready!");
+
+let hacker1 = "Tim"
+console.log("The driver's name is " + hacker1);
+
+let hacker2 = "Saeka"
+ console.log("The navigator's name is " + hacker2);
```

Comment:
This is good! What is usually done is to write this using the dollar signs and backticks. This way you do not need to use the plus sign and your code becomes cleaner. 
e.g.
``console.log(`The driver's name is ${hacker1}`)``